src/hs_analysis_plugins.c

```diff
@@ -67,7 +68,8 @@ static int inject_message(void *parent, const char *pb, size_t pb_len)
         }
       }
     }
-    if (backpressure) {
+    if (backpressure && last_bp_check < time(NULL)) {
```

Comment:
should we just call `time()` once here since only seconds precision is required?

Reply:
Doing it this way we almost never call it.  If we save it off we have to call it for every message even when we aren't backpressuring